src/redux/categories/Categories.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+const CHECK_STATUS = 'bookstore/categories/CHECK_STATUS';
+
+export function checkStatus(category) {
+  return { type: CHECK_STATUS, category };
+}
+
+const initialState = {
+  categories: [],
+  status: '',
+};
+
+export default function categories(state = initialState, action) {
+  switch (action.type) {
+    case CHECK_STATUS:
+      return { ...state, categories: action.categories, status: 'Under construction' };
+    default:
+      return state;
+  }
+}
+
+export const Categories = () => (
+  <div className="container">
+    <button type="button">Check status</button>
+  </div>
```

Comment:
> * [ ]  Case action.type is `CHECK_STATUS` the `categories` function should return a string _"Under construction"_. Please improve upon this as per this module's requirement
 

- [ ] Great job so far, however as requested by the previous reviewer, Make sure the `Check status` working fine and return the `Under construction` message

Reply:
Hi @BenMukebo, 
Thanks for your feedback. Could you please help me to resolve this issue and check the requirements.

Thanks

Reply:
- [ ] Hello @developertariq, As you can see on the screenshort 👇🏻 these are part of the project requiement.

![im2](https://user-images.githubusercontent.com/79058364/214148029-c462a89c-b4a6-4640-8ad4-7104bfdfbe82.png)